p4src/tna/build.sh

```diff
@@ -37,20 +39,37 @@ function base_build() {
   p4c_flags=""
   mkdir -p ${output_dir}
   (
-    time $P4C_CMD --arch tna -g --create-graphs --verbose 2 \
-      -o ${output_dir} -I ${P4_SRC_DIR} \
-      ${OTHER_PP_FLAGS} \
-      ${p4c_flags} \
-      --p4runtime-files ${output_dir}/p4info.txt \
-      --p4runtime-force-std-externs \
-      ${DIR}/fabric_tna.p4
+    if [ "${1}" == "true" ]; then
```

Comment:
Does it need to be a positional argument? @pierventre I think you asked for this, what's the benefit of a positional argument in this case?
```suggestion
    if [ "${SHOW_SENSITIVE_OUTPUT}" == "true" ]; then
```